pkgs/by-name/va/vale-ls/package.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, fetchFromGitHub
+, makeWrapper
+, rustPlatform
+, pkg-config
+, openssl
+, vale
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "vale-ls";
+  version = "unstable-2024-03-13";
+
+  src = fetchFromGitHub {
+    owner = "errata-ai";
+    repo = "vale-ls";
+    rev = "473e16bc88ec48b35e2bd208adc174878c4d5396";
+    hash = "sha256-ywJsnWMc9NSjYjsK6SXdMAQl+hcP+KQ7Xp1A99aeqAg=";
+  };
+
+  nativeBuildInputs = [
+    rustPlatform.bindgenHook
+    pkg-config
+    makeWrapper
+  ];
+
+  buildInputs = [ openssl ];
+
+  # The following tests are reaching to the network.
+  checkFlags = [
+    "--skip=vale::tests"
+  ];
```

Can you something along the following lines, so this package isn't broken on `aarch64`?

```suggestion
  checkFlags = [
    # The following tests are reaching to the network.
    "--skip=vale::tests"
  ] ++ lib.optionals (stdenv.isLinux && stdenv.isAarch64) [
    # This test does not account for the existence of aarch64-linux machines,
    # despite upstream shipping artifacts for that architecture
    "--skip=utils::tests::arch"
  ];
```